include/expansion.h

```diff
@@ -19,7 +19,7 @@
 
 t_list	*expand(t_list *cmd_args, t_ctx *ctx);
 
-char	*expand_variable(char *str, t_ctx *ctx, bool is_smart_expand);
+char	*expand_variable_for_heredoc(char *str, t_ctx *ctx);
```

Comment:
Is `expand_variable_heredoc` a more appropriate naming?

Reply:
True.

Then, is `exec/wait_children.c` more appropriate than`exec/wait_for_children.c` too?

Reply:
I think so 👍🏼

Reply:
I'll handle renaming `wait_for_children` in #137